packages/manager/apps/dedicated/client/app/ip/byoip/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,40 @@
+{
+  "ip_byoip": "BYOIP",
+  "ip_byoip_heading": "Bring Your Own IP (BYOIP)",
```

Comment:
Just for confirmation, it's English required content, right ?

Reply:
Yes since it is product name, it will be like this